courses/models_test.py

```diff
@@ -24,6 +25,15 @@ def test_to_string(self):
         assert "{}".format(prog) == "Title"
 
 
+def from_weeks(weeks, now=None):
+    """Helper function to get a date adjusted by a number of weeks"""
+    if weeks is None:
+        return None
+    if now is None:
+        now = datetime.now(tz=pytz.UTC)
+    return now + timedelta(weeks=weeks)
+
+
```

Comment:
I wanted it to be usable to make data for the parameterized tests, and I figured it was general enough to not be tied to a particular test